packages/rum/src/domain/segmentCollection/segmentCollection.ts

```diff
@@ -197,6 +197,6 @@ export function computeSegmentContext(
   }
 }
 
-export function setMaxSegmentSize(newSize: number = SEND_BEACON_BYTE_LENGTH_LIMIT) {
-  MAX_SEGMENT_SIZE = newSize
+export function setMaxSegmentSize(newSize: number = SEND_BEACON_BYTES_LIMIT) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏rename this function to `setSegmentBytesLimit`

Reply:
Indeed